CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/ViewModels/Operator/OperatorHomeViewModel.cs

```diff
@@ -0,0 +1,79 @@
+﻿using CheckDrive.Mobile.Models;
+using CheckDrive.Mobile.Models.Review;
+using CheckDrive.Mobile.Stores.Operator;
+using CheckDrive.Mobile.Views.Operator;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using Xamarin.Forms;
+
+namespace CheckDrive.Mobile.ViewModels.Operator
+{
+    public class OperatorHomeViewModel : BaseViewModel
+    {
+        private readonly IOperatorStore _operatorStore;
+        public List<OilMark> OilMarks { get; set; }
+        public ObservableCollection<DriverDto> Drivers { get; set; }
+        public Command<DriverDto> ShowReviewPopupCommand { get; }
+        public DateTime CurrentDate { get; }
+
+        public OperatorHomeViewModel()
+        {
+            _operatorStore = DependencyService.Get<IOperatorStore>();
+            CurrentDate = DateTime.Now;
+
+            OilMarks = new List<OilMark>
+            {
+                new OilMark { Id = 1, Name = "80" },
+                new OilMark { Id = 2, Name = "85" },
+                new OilMark { Id = 3, Name = "90" },
+                new OilMark { Id = 4, Name = "92" },
+            };
+            Drivers = new ObservableCollection<DriverDto>
+            {
+                new DriverDto { FullName = "John de Brueny" },
+                new DriverDto { FullName = "Jane Smith"}
+            };
```

Comment:
Those data should come from the mock store so that when you implement real store you don't need to change anything  here.